From f2507bab294de18f64270b33a9638ebc1abba99d Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 14:58:18 -0400
Subject: [infiniband] usnic: Fix typo "Ignorning" -> "Ignoring"

Message-id: <a690f462d8470ed491ffde46aa579f3adcf6e98a.1411569661.git.dledford@redhat.com>
Patchwork-id: 96582
O-Subject: [Patch RHEL7 16/29] IB/usnic: Fix typo "Ignorning" -> "Ignoring"
Bugzilla: 916382
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Roland Dreier <roland@purestorage.com>

Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit c30392ab5bb536fef268c22804dafded15170d14)
Signed-off-by: Doug Ledford <dledford@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/usnic/usnic_ib_main.c b/drivers/infiniband/hw/usnic/usnic_ib_main.c
index f7be445..7d2efd4 100644
--- a/drivers/infiniband/hw/usnic/usnic_ib_main.c
+++ b/drivers/infiniband/hw/usnic/usnic_ib_main.c
@@ -169,7 +169,7 @@ static void usnic_ib_handle_usdev_event(struct usnic_ib_dev *us_ibdev,
    ib_event.element.port_num = 1;
    ib_dispatch_event(&ib_event);
   } else {
-   usnic_dbg("Ignorning %s on %s\n",
+   usnic_dbg("Ignoring %s on %s\n",
      usnic_ib_netdev_event_to_string(event),
      us_ibdev->ib_dev.name);
   }
@@ -177,7 +177,7 @@ static void usnic_ib_handle_usdev_event(struct usnic_ib_dev *us_ibdev,
  case NETDEV_CHANGEADDR:
   if (!memcmp(us_ibdev->ufdev->mac, netdev->dev_addr,
     sizeof(us_ibdev->ufdev->mac))) {
-   usnic_dbg("Ignorning addr change on %s\n",
+   usnic_dbg("Ignoring addr change on %s\n",
      us_ibdev->ib_dev.name);
   } else {
    usnic_info(" %s old mac: %pM new mac: %pM\n",
@@ -206,7 +206,7 @@ static void usnic_ib_handle_usdev_event(struct usnic_ib_dev *us_ibdev,
   }
   break;
  default:
-  usnic_dbg("Ignorning event %s on %s",
+  usnic_dbg("Ignoring event %s on %s",
     usnic_ib_netdev_event_to_string(event),
     us_ibdev->ib_dev.name);
  }
@@ -268,7 +268,7 @@ static int usnic_ib_handle_inet_event(struct usnic_ib_dev *us_ibdev,
   ib_dispatch_event(&ib_event);
   break;
  default:
-  usnic_info("Ignorning event %s on %s",
+  usnic_info("Ignoring event %s on %s",
     usnic_ib_netdev_event_to_string(event),
     us_ibdev->ib_dev.name);
  }
-- 
1.7.1